public/stylesheets/style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
How do I remove a file from git, will it get removed automatically if I commit after deleting it?